packages/cxl-ui/src/components/cxl-marketing-nav-sticky.js

```diff
@@ -0,0 +1,47 @@
+import 'headroom.js';
+import cxlMarketingNavStickyStyles from '../styles/cxl-marketing-nav-sticky-css.js';
+
+const defaultConfig = {
+  stickyElement: '.site-header'
+};
+
+const tmpl = document.createElement('template');
+tmpl.innerHTML = `<style id="cxl-marketing-nav-sticky">${cxlMarketingNavStickyStyles}</style>`;
+document.head.appendChild(tmpl.content);
+
+class CXLMarketingNavStickyElement {
+  constructor(config) {
+    this.config = { ...defaultConfig, ...config };
+
+    this.dropInHeader();
+  }
+
+  dropInHeader() {
+    const headerElement = document.querySelector(this.config.stickyElement);
+
+    const options = {
+      tolerance: 5,
+      offset: headerElement.offsetHeight * 2,
+      // scroller: window,
+      classes: {
+        initial: 'site-header--animated',
+        pinned: 'site-header--pinned',
+        unpinned: 'site-header--unpinned',
+        top: 'site-header--top',
+        notTop: 'site-header--not-top',
+        bottom: 'site-header--bottom',
+        notBottom: 'site-header--not-bottom'
+      },
+      onPin() {},
+
+      onTop() {}
+    };
+
+    /* global Headroom */
+    /* eslint no-undef: "error" */
+    const headroom = new Headroom(headerElement, options);
```

Comment:
Why is our Headroom instance some random dangling variable, instead of being a CXLMarketingNav object property? Something like:

```js
  @property({ type: Object })
  menuGlobalItemsSelectedIdx = {
    instance: null,
    config: {
      ...
    }
  };
```

https://github.com/conversionxl/aybolit/blob/master/packages/cxl-ui/src/components/cxl-marketing-nav.js#L10

Reply:
No we don't, just wanted to keep it separate so easier to debug I guess.

Thanks for the pointer, I didn't consider "Decorators", will keep it in mind for further work.